### ISSUE TITLE
entity.distribution.present check

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-checks#74]
## Description

Check if an entity distribution URL is present and resolvable

## Priority

- FAIR: Required

## Issues
- should every data entity in a dataset be checked for resolvability, as there may be hundreds in a dataset?

## Procedure

- for each entity, extract the entity distribution URL
- check that it is resolvable. If it is not present or not resolvable the check fails and the non-compliant entities are listed

**Comment by gothub** [comment]
Duplicate of https://github.com/NCEAS/metadig-checks/issues/75
